### PR TITLE
[devel-40] Dockerfile: build using RPM packages

### PR DESCRIPTION
**Pull request by vrutkovs**
This should significantly improve build time and decrease image size, as it uses pre-built opensift-ansible RPMs, drops some dependencies (most notably java) and installs less packages via pip

### CODE REVIEW

File: images/installer/Dockerfile [hunk 1]
```diff
@@ -4,40 +4,33 @@ MAINTAINER OpenShift Team <dev@lists.openshift.redhat.com>
...
 
 ENV USER_UID=1001 \
-    HOME=/opt/app-root/src \
+    HOME=/tmp \
     WORK_DIR=/usr/share/ansible/openshift-ansible \
     OPTS="-v"
 
 # Add image scripts and files for running as a system container
 COPY images/installer/root /
-# Include playbooks, roles, plugins, etc. from this repo
```
Comment (michaelgugino) [inline images/installer/Dockerfile]: Are we building this image during a CI run on this repo?  If so, we probably want to keep these copied in from the tree as the rpm bits won't be what we want to test.

Comment (vrutkovs) [reply images/installer/Dockerfile]: CI would first build RPM from the spec and then build this image using a temporary repo. One of the RPMs would be `openshift-test` with `test/` directory

**Comment by mtnbikenc** [comment]
@vrutkovs When I try to build this image locally I get `No package openshift-ansible-test available.`

If this is no longer meant to be used as a containerized openshift-ansible installer and only an image used by CI, all the docs regarding building and using this image should also be removed.

**Comment by vrutkovs** [comment]
@mtnbikenc this change makes local development a bit more complicated, yes - you'd have to build RPMs (one of them would be -test), inject `build` repo in the base image and run the build

However this change makes Dockerfile for CI a bit smaller and ensures spec files are listing all necessary files

**Comment by vrutkovs** [comment]
```
'SSH Error: data could not be sent to remote host "10.0.157.201". Make sure this host can be reached over ssh'
``` in scaled up nodes, not sure if this is a flake or this change doesn't install some necessary component

**Comment by vrutkovs** [comment]
Force pushed to create a new namespace after template has been updated

**Pull request merged**